witchcraft-server/Cargo.toml

```diff
@@ -57,14 +57,14 @@ http-zipkin = "0.4"
 http = "1"
 hyper-util = { version = "0.1", features = ["tokio"] }
 hyper = { version = "1", features = ["http1", "http2", "server"] }
-itertools = "0.13"
+itertools = "0.14"
 lazycell = "1.3"
 libc = "0.2"
 log = "0.4"
-minidump-processor = "0.23"
-minidump-unwind = "0.23"
+minidump-processor = "0.24"
+minidump-unwind = "0.24"
 minidump-writer = "0.10"
-minidump = "0.23"
+minidump = "0.24"
```

Comment:
It seems like dependabot isn't running properly against the repo anymore so I folded these in.